### DISCUSSION TITLE
Validate user emails

### DISCUSSION THREAD

**Discussion by Priscilaboldo** [community/community#50060] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I would like to know how do I validate the emails of each user? Where can I "see" this information?
How can I tell if an organization user is using a personal or corporate account?

**Comment by neutrinomuon** [comment]
Hello,

as a regular user on GitHub, you cannot validate the emails of other users or see the information related to their email validation status. Each user can verify their own email address by going to their account settings and clicking on "Emails" in the left-hand menu. From there, they can add, verify, or remove email addresses associated with their account.

If you are an owner or administrator of an organization on GitHub, you can see the email addresses of members in the organization's settings. To do this, go to the organization's page, click on "Settings," and then click on "Members" in the left-hand menu. From there, you can see the email addresses of all the members of the organization.

However, you will not be able to see whether a user is using a personal or corporate account based on their email address alone. Users can use any email address to sign up for a GitHub account, and it is up to them to indicate whether they are using a personal or corporate account in their profile information. If you need to know whether a user is using a corporate account for organizational purposes, you may need to ask them directly or check with your organization's IT department.

I hope this helps
